ArduPlane/Plane.h

```diff
@@ -231,6 +232,7 @@ class Plane : public AP_HAL::HAL::Callbacks {
 
     AP_TECS TECS_controller{ahrs, aparm, landing};
     AP_L1_Control L1_controller{ahrs, &TECS_controller};
+    AP_LQR_Control LQR_controller{ahrs, &TECS_controller};
```

Comment:
You need to remove this one and initialize the one in ParametersG2. The commit I mentioned also shows how to initialize.

Reply:
Oh ok. Done.